stories/core/Table.stories.tsx

```diff
@@ -1320,6 +1322,7 @@ export const Full: Story<Partial<TableProps>> = (args) => {
               </>
             ),
           },
+          ActionColumn({ hasColumnManager: true }),
```

Comment:
I would say to marge this column with `click-me` column but better change content to triple dots dropdown menu with some mocked actions like edit, delete. Because it is the whole point of having `ActionColumn` as a column and not regular prop in Table.

Reply:
Changed this column to an Action Column whose cells are triple dot dropdown menus